templates/VALIDATION.md

```diff
@@ -51,6 +51,18 @@ If a rule is meaningless, for example if it has no arguments (see below), its be
 ### JSONPaths
 
 For every jsonpath mentioned in this document, unless specified otherwise, the root is the objects: element of the template.
+Unless otherwise specified, the jsonpath must be enclosed in double brackets - this is to disambiguate plain strings from jsonpaths.
+Example:
+
+good:
+```
+{{.spec.domain.resources.requests.memory}}
```

Comment:
This is problematic. Double brackets are used in ansible and we use ansible to generate the final template files. We would create escaping hell this way..

Reply:
No big deal, I just need a clean and simple way to dfistinguish what is supposed to be a jsonpath and what must be treated as string literal. Any syntax is fine. Maybe

  jsonpath::.spec.domain.resources.requests.memory

? aka "jsonpath::" prefix